src/plotman/plotman.py

```diff
@@ -30,7 +30,16 @@ def parse_args(self):
  
         sp.add_parser('dirs', help='show directories info')
 
-        sp.add_parser('interactive', help='run interactive control/monitoring mode')
+        p_interactive = sp.add_parser('interactive', help='run interactive control/monitoring mode')
+        p_interactive.add_argument('--autostart-plotting',
+            action='store_true',
+            default=None,
+            help='Automatically start plotting when the tool opens')
+
+        p_interactive.add_argument('--no-autostart-plotting',
+            action='store_true',
+            default=None,
+            help='Do not start plotting when the tool opens')
```

Comment:
Take a look at `BooleanOptionalAction` in https://docs.python.org/3/library/argparse.html#action.  I think this can be a single added argument.

Reply:
Slight update to account for the not-passed case so we can use the value in the config file.

https://replit.com/@altendky/ImmaterialTintedInterfacestandard-2

```python
import argparse


parser = argparse.ArgumentParser()
parser.add_argument('--x', action='store_true', default=None, dest='x')
parser.add_argument('--no-x', action='store_false', default=None, dest='x')

print(parser.parse_args([]).x, None)
print(parser.parse_args(['--x']).x, True)
print(parser.parse_args(['--no-x']).x, False)
print(parser.parse_args(['--x', '--no-x']).x, False)
print(parser.parse_args(['--no-x', '--x']).x, True)
```

```
None None
True True
False False
False False
True True
```